deploy/manifests/dev/us-east-2/tenant/storetheindex/assigner/kustomization.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+namespace: storetheindex
+
+resources:
+  - ../../../../../base/assigner
+  - ingress.yaml
+
+patchesStrategicMerge:
+  - deployment.yaml
+
+secretGenerator:
+  - name: identity
+    behavior: replace
+    files:
+      - identity.key=indexer-0-identity.key.encrypted # 12D3KooWDBjcDRQ7CKJeF9Yy3UKbriHfyETDrXKzTDB6biH3ibBd
```

Comment:
As is this won't work because the name of the file is actually `[indexer-0-identity.encrypted](https://github.com/filecoin-project/storetheindex/pull/1035/files#diff-e2ea8bf2f806f24ec0511e2202beabaf9beaf191af83491dc97b37691c1968b4)`. See earlier comment on renaming this.